documentation/library-reference/source/language-extensions/string-literals.rst

```diff
@@ -0,0 +1,18 @@
+String Literal Syntax
+=====================
+
+`Dylan Enhancement Proposal (DEP) 12
+<https://opendylan.org/proposals/dep-0012-string-literals.html>`_ extends the
+literal syntax for strings to include strings without any escape character
+processing ("raw" strings) and multi-line strings. Briefly,
+
+#. Multi-line strings begin with three double-quote characters: ``"""``
+
+#. End-of-line sequences in multi-line strings are always parsed as a single
+   Newline (``\\n``) character, regardless of operating system.
```

Comment:
regardless of source file line endings or the conventions of the operating system.

Reply:
Done